src/features/chat/Chats.js

```diff
@@ -12,12 +16,50 @@ const StyledChats = styled.div`
   background-color: black;
 `;
 
+const BASE_URL = process.env.REACT_APP_AXIOS_BASE_URL;
+
 function Chats() {
-  // const navigate = useNavigate();
+  const navigate = useNavigate();
+  const userId = useSelector((state) => state.login.userId);
+  const [chatrooms, setChatrooms] = useState([]);
+
+  async function getToilet(toiletId) {
+    const toilet = await axios.get(`${BASE_URL}/toilets/toilet/${toiletId}`);
+
+    return toilet;
+  }
+
+  function findLastChat(chatList) {
+    const lastChat = chatList[chatList.length - 1];
+
+    return lastChat;
+  }
+
+  useEffect(() => {
+    async function findMyChatRoooms() {
+      const myChatrooms = await axios.get(
+        `${BASE_URL}/chatroom/live-chatroom/${userId}`
+      );
+
+      setChatrooms(myChatrooms);
+    }
+
+    if (userId) findMyChatRoooms();
+  }, [userId]);
 
   return (
     <StyledChats>
-      <HeaderSub />
+      <HeaderSub onClick={() => navigate("/chats")} />
+      {chatrooms &&
+        chatrooms.map((chatroom) => (
+          <>
+            <Title title={getToilet(chatroom.toilet).toiletName} />
+            <List2Lines
+              label={findLastChat(chatroom.chatList).sender}
+              secondary={findLastChat(chatroom.chatList).message}
```

Comment:
아마 제가 이해한게 맞다면 label 이 리스트의 h1 과 같은 주 제목, secondary 가 소 제목 인거 같아요. 개인적으로는 secondary라는 단어로 해당 내용을 파악하기 힘들 수 있을거 같아서 더 명확한 네이밍이 있으면 좋을 거 같네요. 아래와 같은 네이밍은 어떨까요.

```suggestion
              mainLabel={findLastChat(chatroom.chatList).sender}
              subLabel={findLastChat(chatroom.chatList).message}
```